src/sql/parts/dashboard/dashboardConfig.contribution.ts

```diff
@@ -4,12 +4,13 @@
  *--------------------------------------------------------------------------------------------*/
 import { Registry } from 'vs/platform/registry/common/platform';
 import { IConfigurationRegistry, Extensions, IConfigurationNode } from 'vs/platform/configuration/common/configurationRegistry';
+import { DASHBOARD_CONFIG_ID } from 'sql/parts/dashboard/pages/dashboardPagecontribution';
```

Comment:
This line is causing the Linux build to fail - paths are case sensitive there and you lowercases `contribution`